lib/ecto/adapters/postgres/connection.ex

```diff
@@ -250,6 +250,27 @@ if Code.ensure_loaded?(Postgrex) do
       ["DELETE FROM ", quote_table(prefix, table), " WHERE ", filters | returning(returning)]
     end
 
+    @impl true
+    def explain_query(query, opts \\ []) do
+      verbose = Keyword.get(opts, :verbose, false) |> expr(nil, nil)
+      costs   = Keyword.get(opts, :costs,   true)  |> expr(nil, nil)
+      buffers = Keyword.get(opts, :buffers, false) |> expr(nil, nil)
+
+      analyze =
+        case buffers do
+          "TRUE" -> "TRUE"
+          "FALSE" -> Keyword.get(opts, :analyze, false) |> expr(nil, nil)
```

Comment:
Instead of calling `expr(nil, nil)`, move the clauses that you want from `expr` to a separete function called "quote_values", that handle values exclusively. I am assuming those are all booleans, so you can simply call it quote_boolean(...).